### PR TITLE
Feature/tilesupdate

### PR DESCRIPTION
**Pull request by MagicAardvark**
feat(Tiles): Adding default value, further enhanced styling to Tiles component

##### **What did you change?**
Added default value to Tiles component by passing in value a tile value can be set
changed Tiles styling after review with @bkimball
change readme.md file for Tiles

##### **Reviewers**
* @jgodi @bvkimball @krsween 

##### **Checklist (completed via merger)**
* [x] Changes are limited to a single goal (no scope creep)
* [x] Code can be automatically merged (no conflicts)
* [x] Code follows the standards laid out in the [contributing guide](https://github.com/bullhorn/novo-elements/blob/master/CONTRIBUTING.md)
* [x] Passes all existing automated tests
* [x] New functions include new tests
* [x] New functions are documented (with a description, list of inputs, and expected output)
* [x] Visually tested in supported browsers and devices

### CODE REVIEW

File: src/elements/form/extras/tiles-input/TilesInput.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        NOVO_PICKER_ELEMENTS,
+        NgModel
+    ],
+    template: `
+        <i *ngIf="required" class="required-indicator" [ngClass]="{'bhi-circle': !control.valid, 'bhi-check': control.valid}"></i>
+        <novo-tiles [options]="options" (changed)="onChanged($event)" [value]="value"></novo-tiles>
+        <span class="error-message" *ngIf="required && control.touched && control?.errors?.required">{{labels.required}}</span>
+    `
+})
+export class TilesInput extends BaseInput {
```
Comment (krsween) [inline src/elements/form/extras/tiles-input/TilesInput.js]: If we have this here, we likely don't need the manual implementation in the tiles component.

File: src/elements/form/extras/tiles-input/TilesInput.spec.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
+// // import {DOM} from 'angular2/src/platform/dom/dom_adapter';
```
Comment (krsween) [inline src/elements/form/extras/tiles-input/TilesInput.spec.js]: We can work on this together.

File: src/elements/tiles/README.md [hunk 3]
```diff
@@ -10,4 +10,6 @@ A tile is like a radio button group.
 - `'options' : Array`
     * Collection of values to use
 - `'required' : String`
-    * Is this field required
\ No newline at end of file
+    * Is this field required
+- `'value' : String`
```
Comment (jgodi) [inline src/elements/tiles/README.md]: ngModel now?

File: src/elements/tiles/Tiles.js [hunk 4]
```diff
@@ -1,49 +1,43 @@
...
+import { CORE_DIRECTIVES, NgControl, NgModel } from '@angular/common';
 
 @Component({
     selector: 'novo-tiles',
     inputs: [
         'name',
         'options',
-        'required'
+        'required',
+        'value'
```
Comment (jgodi) [inline src/elements/tiles/Tiles.js]: Shouldnt need value with ngModel

File: src/elements/form/extras/tiles-input/TilesInput.js [hunk 5]
```diff
@@ -33,8 +33,9 @@ export class TilesInput extends BaseInput {
         this.labels = labels;
     }
 
-    onChanged(e) {
-        this.value = e.value ? e.value : null;
```
Comment (krsween) [inline src/elements/form/extras/tiles-input/TilesInput.js]: This is a question: do we want the value or the object's value property at this level?

Comment (jgodi) [inline src/elements/form/extras/tiles-input/TilesInput.js]: From tiles it just emits the string value right?

File: src/elements/tiles/Tiles.js [hunk 6]
```diff
@@ -71,8 +64,21 @@ export class Tiles {
             option.checked = false;
         }
         item.checked = !item.checked;
-        if (this.update) this.update.emit(item.value);
-        if (this.control) this.control.updateValue(item.value);
+        this.changed.emit(item.value);
```
Comment (jgodi) [inline src/elements/tiles/Tiles.js]: @krsween here

Comment (krsween) [inline src/elements/tiles/Tiles.js]: Right, but here we could emit `item`. I wasn't sure if there's a preference on how/where that gets parsed by the other components.

Comment (jgodi) [inline src/elements/tiles/Tiles.js]: Gotcha.. I am fine with either way.

**Pull request merged**